rosdep/python.yaml

```
@@ -7230,6 +7230,15 @@ python3-pkg-resources:
   opensuse: [python3-setuptools]
   rhel: ['python%{python3_pkgversion}-setuptools']
   ubuntu: [python3-pkg-resources]
+python3-ply:
+  arch: [python-ply]
+  debian: [python3-ply]
+  fedora: [python3-ply]
+  gentoo: [dev-python/ply]
+  opensuse: [python3-ply]
+  rhel:
+    "8": [python3-ply]
```

Please use forward-looking syntax, as is the pattern used for the many other instances in rosdep where a package is available for the newest release but not older releases.

Also, please single-quote strings, as is the pattern used throughout rosdep.

```suggestion
    '*': [python3-ply]
    '7': null
```